src/poll.rs

```diff
@@ -1,10 +1,26 @@
 use crate::{event, sys, Events, Interest, Token};
 use log::trace;
-#[cfg(unix)]
-use std::os::unix::io::{AsRawFd, RawFd};
 use std::time::Duration;
 use std::{fmt, io};
 
+#[cfg(all(
```

Comment:
I wouldn't repeat this `cfg` block so often; I'd follow along with what the rest of `mio` does and create a macro that encapsulates the `cfg`.